chore(deps): bump dependencies

mokurin000 opened the pull request.

mpizenberg commented:
Thanks for the upgrades, I’ll update the CI and see if it still runs afterwards.

mpizenberg commented:
It seems the upgraded nalgebra does not like the existing code. Can you have a look at it please.

mokurin000 commented:
> It seems the upgraded nalgebra does not like the existing code. Can you have a look at it please.

Strange though.. I did build it on my machine previously, and running without issue

Oh maybe I did not enable such feature

mokurin000 commented:
@mpizenberg Hi, I did migration for nalgebra, modified some generic type for image-rs, replaced deperecated function name alias

```text
running 4 tests
test nalgebra::tests::test_identical_par_idct_result ... ok
test slice::tests::test_identical_par_idct_result ... ok
test slice::tests::test_identical_par_dct_result ... ok
test nalgebra::tests::test_identical_par_dct_result ... ok
```

mpizenberg commented:
Nice! Why does Apple always have to get in the way!?? XD

mpizenberg commented:
The issue seems to be related to this https://github.com/robohouse-delft/show-image-rs/issues/54
So I just pinned show-image to v0.14.0
